### ISSUE TITLE
Get the folder name and the request name in a variable

### ISSUE THREAD

**Issue by NPunzo** [usebruno/bruno#3507]
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

### Describe the feature you want to add

This is a feature I've found in Insomnia that I loved because it allowed me to industrialize REST API testing on a very large project.
The goal is to allow to get the folder name and the request name in a variable, making possible to create massively new requests really easily, just while duplicating an existing resource and renaming it.

**Example of tree structure :**
- **my-awesome-api** <-- folder
  - /api/v1/my/awesome/resource
  - /api/v2/my/awesome/resource
  - /api/v2/my/awesome/other/resource 

So, if we get into the request called "/api/v1/my/awesome/resource", the URL will be : 
_{{baseUrl}}{{**folder.name**}}{{**request.name**}}_
If we look into the request named "/api/v2/my/awesome/resource", the URL will also be : 
_{{baseUrl}}{{**folder.name**}}{{**request.name**}}_
And for the request named "/api//v2/my/awesome/other/resource", the URL will, again, be : 
_{{baseUrl}}{{**folder.name**}}{{**request.name**}}_

Imagine what you can do on a large project with endpoints sharing the same structure ;) 

It requires also to allow slashes in the resource names. see #3266

### Mockups or Images of the feature

Here is an example of how I've made it in Insomnia :
![image](https://github.com/user-attachments/assets/9422e952-c288-49ce-9b1a-0ea0d22591f6)

**Comment by NPunzo** [comment]
Up

**Comment by ramonlucas** [comment]
I've been waiting for this feature for so long...
It's so simple...

**Comment by sreelakshmi-bruno** [comment]
Hi, we do support getting a  req name via the API `req.getName()`(See [docs](https://docs.usebruno.com/testing/script/javascript-reference#req-getname)). Support for accessing the folder name via the API is also planned.

**Comment by ramonlucas** [comment]
> Hi, we do support getting a req name via the API `req.getName()`(See [docs](https://docs.usebruno.com/testing/script/javascript-reference#req-getname)). Support for accessing the folder name via the API is also planned.

No, you support the only the current folder name, but not the full patch name.
What we want is to get the full patch.
It is also described here: https://github.com/usebruno/bruno/issues/3817
